media/js/firefox/australis/growth-firstrun-test1.js

```diff
@@ -0,0 +1,258 @@
+;(function($, Mozilla) {
+    'use strict';
+
+    // declare vars (might as well be here because hoisting)
+    var $body;
+    var $ghosteryIframe;
+    var $outerWrapper = $('#outer-wrapper');
+    var pageId = $('body').prop('id');
+
+    // determines if tour opens or not
+    var showDoorhanger;
+
+    var gaVariation;
+
+    // reference FxA elements
+    var $fxaOverlay;
+    var $fxaIframe;
+    var $fxaExtra;
+    var $fxaValueProp;
+    var $fxaDelayedCTAs;
+
+    // store client ID for FxA integration
+    var fxaClient;
+    var fxaClientID;
+
+    // function expression vars
+    var initFxA;
+    var enableIframe;
+    var showFxA;
+    var trackSyncClick;
+
+    var toggleDefaultContent = function(show) {
+        // show default #outer-wrapper
+        if (show) {
+            $outerWrapper.show();
+
+            Mozilla.syncAnimation();
+        } else {
+            $outerWrapper.hide();
+        }
+    };
+
+    // fallback in case UITour doesn't load
+    $outerWrapper.show();
+
+    // Only run experiment content if user is on Firefox 35 for desktop and has not just submitted email form
+    if (window.isFirefox() && !window.isFirefoxMobile() && window.getFirefoxMasterVersion() >= 35 && window.location.hash !== '#footer-email-form') {
+        // initialize vars
+        $body = $('body');
+        $ghosteryIframe = $('iframe.addon');
+
+        // grab FxA settings from data-* attributes
+        $fxaExtra = $('#fxa-extra');
+
+        Mozilla.FXA_RELIER = {
+            clientId: $fxaExtra.attr('data-fxa-client-id'),
```

Comment:
Nit: data attributes should be read as per the spec/jQuery docs: http://api.jquery.com/data/#data-html5

e.g. `$fxaExtra.data('fxaClientId')`